bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -584,7 +591,11 @@ LedgerIdGenerator getLedgerIdGenerator() {
      * report fake bytes with a mathching MAC unless it knows the password
      */
     public enum DigestType {
-        MAC, CRC32
+        MAC, CRC32;
+
+        public static DigestType fromApiDigestType(org.apache.bookkeeper.client.api.DigestType digestType) {
```

Comment:
hmm..again it is confusing to have same enum names 

Bookkeeper.DigestType and client.api.DigestType

Reply:
we want new clients not to use (now) legacy org.apache.bookkeeper.client package so we need new enums and classes, using the same name in this case is ok as we are using different packages